### PR TITLE
Lost pwd

### LINKED ISSUE #28: Lost-pwd
form process & template
Sending mail

Estimation : 3h30

### PR DESCRIPTION
**Pull request by LFZDavid**

### CODE REVIEW

File: src/Controller/SecurityController.php [hunk 1]
```diff
@@ -95,18 +96,15 @@ public function edit(Request $request, EntityManagerInterface $manager, AvatarFi
...
-            /** @var UploadedFile $imgFile */
             $imgFile = $form->get('avatar')->getData();
-
             if ($file = $form->get('avatar')->get('file')->getData()) {
                 if($user->getAvatar()->getUrl()) {
                     $fileUploader->deleteAvatarFile($user->getAvatar());
                 }
                 $imgFileName = $fileUploader->upload($file);
                 $imgFile->setUrl($imgFileName);
+                $manager->persist($imgFile);
```
Comment (crazymad) [inline src/Controller/SecurityController.php]: cascade persist ?

File: src/Controller/SecurityController.php [hunk 2]
```diff
@@ -116,5 +114,47 @@ public function edit(Request $request, EntityManagerInterface $manager, AvatarFi
...
+        $form->handleRequest($request);
+        if($form->isSubmitted() && $form->isValid()) {
+            $submitedName = $form->get('username')->getData();
+            $user = $userRepository->findOneBy([
+                'name' => $submitedName,
+                'active' => true,
+            ]);
+
+            if(!$user) {
+                // todo : message user doesn't exist or not active
```
Comment (crazymad) [inline src/Controller/SecurityController.php:136]: when todo ?

File: tests/Controller/SecurityTest.php [hunk 3]
```diff
@@ -118,4 +118,17 @@ public function testUpdateUser()
...
     }
     
+
+    public function testGetForgotPwdForm()
+    {
+        $crawler = $this->client->request('GET', '/security/lost_pwd');
+        $this->assertResponseIsSuccessful();
+    }
+    /**
+     * todo:
```
Comment (crazymad) [inline tests/Controller/SecurityTest.php]: ?

**Review by crazymad — COMMENTED** [review]
Do you plan to do the TODO in a next iteration or did you forgot ?

File: src/Controller/SecurityController.php (hunk 2 above)
Comment (LFZDavid) [reply src/Controller/SecurityController.php:136]: On flash-messages Issue

File: src/Controller/SecurityController.php (hunk 1 above)
Comment (LFZDavid) [reply src/Controller/SecurityController.php]: fixed in lost-password branch

File: tests/Controller/SecurityTest.php (hunk 3 above)
Comment (LFZDavid) [reply tests/Controller/SecurityTest.php]: forgot to delete comments. fixed in lost-password branch

**Pull request merged**